spec/game_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "./lib/game"
+
+RSpec.describe Game do
+
+  it 'exists and has readable attributes' do
+    stats = {
+            game_id: "2012030221",
+            season: "20122013",
+            away_team_id: "3",
+            home_team_id: "6",
+            away_goals: "2",
+            home_goals: "3"
+            }
+    game = Game.new(stats)
```

Comment:
This will be really helpful to shorten some of our existing methods with refactoring, especially as we're currently reading the CSV raw data each time